### PR TITLE
feat: add new property isInvertedRowItem to invert row items

### PR DESCRIPTION
**Pull request by JeffGuKang**
Make able to invert row items. 

ex)
```ts
<SectionGrid
  invertedRow
  ...
/>
```
[[0, 1, 2], [0, 1, 2], [0, 1, 2]]
=>
[[2, 1, 0], [2, 1, 0], [2, 1, 0]]

### CODE REVIEW

File: SectionGrid.js [hunk 1]
```diff
@@ -23,6 +22,7 @@ const SectionGrid = memo(
       onLayout,
       additionalRowStyle: externalRowStyle,
       itemContainerStyle,
+      isInvertedRowItem,
```
Comment (saleel) [inline SectionGrid.js:25]: How about calling this prop `invertedRow`?
We are inverting the row and not the rowItem. Also `is` prefix for booleans are not used by FlatList and SectionList (`horizontal` prop for example) so we can stick to that convention.

File: SectionGrid.js [hunk 2]
```diff
@@ -69,6 +69,7 @@ const SectionGrid = memo(
         separators,
         isFirstRow,
         containerStyle,
+        isInverted,
```
Comment (saleel) [inline SectionGrid.js:72]: We can use the same key name here as well.

File: SectionGrid.js [hunk 3]
```diff
@@ -128,7 +133,12 @@ const SectionGrid = memo(
 
     const groupSectionsFunc = useCallback(
       (section) => {
-        const chunkedData = chunkArray(section.data, itemsPerRow);
+        let chunkedData = chunkArray(section.data, itemsPerRow, true);
```
Comment (saleel) [inline SectionGrid.js:136]: I am not sure why we need the third parameter (`true`) here.

**Review by saleel — CHANGES_REQUESTED** [review]
Looks Good.

I have added some minor comments. Also appreciate if you could add the same prop to `FlatGrid` and update the README file (props table).

Thanks

File: SectionGrid.js (hunk 1 above)
Comment (JeffGuKang) [reply SectionGrid.js:25]: Thanks for good naming. I will apply it.

File: SectionGrid.js (hunk 3 above)
Comment (JeffGuKang) [reply SectionGrid.js:136]: It is a useless parameter. I will remove it.

**Comment by JeffGuKang** [comment]
Squash will be better for applying than merge. :)

**Comment by JeffGuKang** [comment]
@saleel I did commit more :)

**Review by saleel — APPROVED** [review]
Looks good. It would be nice to have the same for FlatGrid as well. I will add that separately.

**Comment by saleel** [comment]
I have made one small change to fix a lint issue, and also added this prop for FlatGrid.

Published version `4.2.0` to npm.
Thanks

**Pull request merged**